src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -330,15 +376,15 @@ public void Args_Script1()
 ", runner.Console.Out.ToString());
         }
 
-        [Fact]
+        [ConditionalFact(typeof(WindowsOnly)), WorkItem(15860, "https://github.com/dotnet/roslyn/issues/15860")]
```

Comment:
What changed that necessitated disabling these tests?  The referenced issue doesn't have any information on this.